modules/custom/d_content/config/update/d_content.8118.yml

```diff
@@ -0,0 +1,12 @@
+core.entity_form_display.node.content_page.default:
+  add:
+    content:
+      field_header_layout:
+        region: content
+        type: options_select
+field.field.node.content_page.field_header_layout:
+  change:
+    expected:
+      required: false
+    new:
+      required: true
```

Comment:
@bartek-flp ten config dopiero w 2.2 wchodzi + mamy już update 8116, który wciąga config ORAZ ma update do wciagania zmian:
![image](https://user-images.githubusercontent.com/37533838/88645745-c5dd3f00-d0c4-11ea-8a8a-ef9e07edd8b9.png)
więc tą zmianę wystarczy wprowadzić w pliku (nie musimy pod to update tworzyć kolejny:
![image](https://user-images.githubusercontent.com/37533838/88645822-da213c00-d0c4-11ea-8040-1f90f212f898.png)